### PR TITLE
Created To-Do List

### PR DESCRIPTION
**Pull request by whid-bb**
### Added CRUD functionality:
 - add item using form
 - change/delete item by clicking on To Do List text
 
### Added remove completed tasks

[Live Demo](https://whid-bb.github.io/to-do-list-review/)

### CODE REVIEW

File: src/js/Ls.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import * as globals from './globals.js';
+
+class Ls {
```
Comment (yohannesdagnachew) [inline src/js/Ls.js:3]: Kindly make your class name descriptive it will make your code clean

**Comment by yohannesdagnachew** [comment]
### Hi @whid-bb 👋 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
### To Highlight 🎉
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️
### Required Changes ♻️
- Kindly consider adding an h1 to index.html as it is more semantic

File: src/js/Ls.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  addToLS = (obj) => {
```
Comment (yohannesdagnachew) [inline src/js/Ls.js]: Kindly make your class name descriptive that makes your code clean

Comment (whid-bb) [reply src/js/Ls.js]: Thank you! I will change that in a moment :)

**Comment by whid-bb** [comment]
@yohannesdagnachew  Thank you for the review!

File: src/js/Ls.js (hunk 2 above)
Comment (whid-bb) [reply src/js/Ls.js]: @yohannesdagnachew Changes have been made :)

**Pull request merged**